pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -104,13 +126,18 @@ class RichProgressBar(ProgressBarBase):
 
     Args:
         refresh_rate: the number of updates per second, must be strictly positive
+        theme: Contains styles used to stylize the progress bar.
 
     Raises:
         ImportError:
             If required `rich` package is not installed on the device.
     """
 
-    def __init__(self, refresh_rate: float = 1.0):
+    def __init__(
+        self,
+        refresh_rate: float = 1.0,
+        theme: RichProgressBarTheme = RichProgressBarTheme(),
+    ):
```

Comment:
```suggestion
    ) -> None:
```
I guess mypy doesn't run on these files :)